Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1,14 +1,933 @@
+
+### Control Settings 
+```json 
+{
+    "NoOfClassicAdminsLimit": 2
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> The count of classic administrators does not exceed 2.
+> 
+> **Failed:** 
+> More than 2 classic administrators accounts found.
+> 
+> **Verify:** 
+> RBAC result not found (sufficient data is not available for evaluation).
+> 
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 You need to remove any 'Classic Administrators/Co-Administrators' who should not be in the role. Please follow these steps: <br />(a) Logon to https://portal.azure.com/ <br />(b) Navigate to Subscriptions <br />(c) Select the subscription <br />(d) Go to 'Access Control (IAM)' and select the 'Classic Administrators' tab. <br />(e) Select the co-administrator account that has to be removed and click on the 'Remove' button. <br />(f) Perform this operation for all the co-administrators that need to be removed from the subscription. 
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list classic role assignment at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Authorization/classicAdministrators?api-version=2015-06-01<br />
+**Properties:** properties.role
+ <br />
+
+<br />
+
+___ 
+
+## Azure_Subscription_AuthZ_Remove_Management_Certs 
+
+### DisplayName 
+Do not use management certificates 
+
+### Rationale 
+Just like classic admins, management certificates were used in the v1 model for script/tool based automation on Azure subscriptions. These management certificates are risky because the (private) key management hygiene tends to be lax. These certificates have no role to play in the current ARM-based model and should be immediately cleaned up if found on a subscription. (VS-deployment certificates from v1 timeframe are a good example of these.) 
+
+### Control Spec 
+
+> **Passed:** 
+> ASC assessment status is healthy.
+> 
+> **Failed:** 
+> ASC assessment status is unhealthy. (or) ASC assessment status is "NotApplicable" with "cause" as either "OffByPolicy" or "Exempt".
+> 
+> **Verify:** 
+> ASC assessment status is not applicable (with "cause" other than "OffByPolicy" and "Exempt"), OR ASC assessment status was not found.
+> 
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 You need to remove any management certificates that are not required. Please follow these steps: <br />(a) Logon to https://portal.azure.com/ <br />(b) Navigate to Subscriptions <br />(c) Select the subscription <br />(d) Go to Settings tab  --> Management Certificates tab --> Delete unwanted management certificates. 
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list security assessments at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Security/assessments?api-version=2020-01-01 <br />
```

Comment:
Please mention the assessment used to evaluate this Control (and, optionally, its UUID).